src/server/services/wms/qgswmsrendercontext.cpp

```diff
@@ -292,7 +292,8 @@ QList<QgsMapLayer *> QgsWmsRenderContext::layersFromGroup( const QString &nickna
   QList<QgsMapLayer *> layers;
   if ( mLayerGroups.contains( nickname ) )
   {
-    for ( QgsMapLayer *layer : mLayerGroups[nickname] )
+    const QList<QgsMapLayer *> layers = mLayerGroups.value( nickname );
+    for ( QgsMapLayer *layer : layers )
```

Comment:
Looked again after the failed tests. Couldn't this be much simpler?

```
  return mLayerGroups.value( nickname );
```

Reply:
something like that
```
QList<QgsMapLayer *> QgsWmsRenderContext::layersFromGroup( const QString &nickname ) const
{
  QList<QgsMapLayer *> layers;
  if ( mLayerGroups.contains( nickname ) )
  {
    layers = mLayerGroups.value( nickname );
  }
  return layers;
}
```

Reply:
```
QList<QgsMapLayer *> QgsWmsRenderContext::layersFromGroup( const QString &nickname ) const
{
  return mLayerGroups.value( nickname );
}
```

Reply:
done thanks